src/qtiCommonRenderer/renderers/interactions/ExtendedTextInteraction.js

```diff
@@ -427,15 +428,29 @@ function inputLimiter(interaction) {
                 1114202, // Ctrl + z
                 1114200 // Ctrl + x
             ];
-            const triggerKeyCodes = [
+            const spaceKeyCodes = [
                 32, // space
                 13, // enter
                 2228237 // shift + enter in ckEditor
             ];
-            let cke;
             let isComposing = false;
             let hasCompositionJustEnded = false;
 
+            const acceptKeyCode = keyCode => _.contains(ignoreKeyCodes, keyCode);
```

Comment:
**suggestion:** to use a [native alternative](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore#_includes) instead of _